pkg/api/grpcServer.go

```diff
@@ -218,6 +218,17 @@ func (s *grpcServer) Check(legacyCtx legacyContext.Context, req *mixerpb.CheckRe
 					GrantedAmount: qma.Amount,
 				}
 			}
+
+			if glog.V(2) {
```

Comment:
do we want this section of code?  Doesn't this imply dimensions that might not be on the quota descriptor?  For instance, I might not choose to dimension quota with `source.name`.

Reply:
regardless of quota dimensioning, I want accessLog to convey just some reasonable notion of source and target, how much quota was requested and how much was granted.

Reply:
+1, I don't like this chunk. _If_ we want this kind of logging, just log "allocated %d tokens along dimensions: %v". Don't hard code attributes anywhere.

Reply:
I just log `destination.service` along with quota response now.

I also log full dimensions at log level 2.